torch/csrc/distributed/rpc/tensorpipe_agent.cpp

```diff
@@ -798,7 +798,7 @@ void TensorPipeAgent::respond(std::shared_ptr<tensorpipe::Pipe>& pipe) {
 
           std::shared_ptr<JitFuture> futureResponseMessage;
           try {
-            futureResponseMessage = cb_->operator()(requestMessage);
+            futureResponseMessage = cb_->operator()(requestMessage, ctx);
```

Comment:
nit: shall we add a comment here to explain why we need to pass ctx to `operator()()`?

Reply:
addressed